src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -37,4 +44,18 @@ public int getBarteringLootChance() {
         return chance.getValue();
     }
 
+    @Nonnull
+    @Override
+    public ItemUseHandler getItemHandler() {
+        return this::use;
+    }
+
+    public void use(PlayerRightClickEvent e) {
+        if (e.getClickedBlock().isPresent()) {
+            if (Tag.SIGNS.isTagged(e.getClickedBlock().get().getType())) {
```

Comment:
Calling the method twice is a bit non-optimal, you should store the Optional<Block> as a variable and use that.